src/Functions/FunctionsJSON.cpp

```diff
@@ -1332,13 +1383,69 @@ class JSONExtractRawImpl
 
     static bool insertResultToColumn(IColumn & dest, const Element & element, std::string_view)
     {
-        ColumnString & col_str = assert_cast<ColumnString &>(dest);
-        auto & chars = col_str.getChars();
-        WriteBufferFromVector<ColumnString::Chars> buf(chars, AppendModeTag());
+        if (dest.getDataType() == TypeIndex::LowCardinality)
+        {
+            ColumnString::Chars chars;
+            WriteBufferFromVector<ColumnString::Chars> buf(chars, AppendModeTag());
+            chars.push_back(0);
+            traverse(element, buf);
+            buf.finalize();
+            std::string str = reinterpret_cast<const char *>(chars.data());
+            chars.push_back(0);
+            assert_cast<ColumnLowCardinality &>(dest).insertData(str.data(), str.size());
```

Comment:
Why do we have first `chars.push_back(0);`? I think we can use here `WriteBufferFromOwnString` instead of using `WriteBufferFromVector` with `ColumnString::Chars` and `str`.

Reply:
I left using chars and removed the str